src/components/Video/style.scss

```diff
@@ -41,16 +41,16 @@
   top: 0;
   background-repeat: no-repeat;
   background-size: contain;
-  width: 26*@unit;
-  height: 26*@unit;
+  width: 26*$unit;
+  height: 26*$unit;
 }
 
-.speak_out_loudIcon {
```

Comment:
You mentioned you don't want to change class names? The old class name should be all fine by the new regex set?

Reply:
I'm fine with that, just thought that this one seemed to be different from all the existing classes & wasn't sure if a regex could handle this mix of styles.